package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Call tracking using Twilio, TwiML and Express.js",
   "main": "index.js",
   "scripts": {
-    "test": "export NODE_ENV=test && mocha test"
+    "start":"node index",
+    "test": "NODE_ENV=test ./node_modules/.bin/mocha --timeout 4000 test"
```

Comment:
I believe the other env vars are loaded either in `.travis.yml` or in `.env` file. 

Regarding the timeout, I believe the default was 2000 ms, which I found restrictive occasionally so I increased it in the npm test script to be sure it won't cause the tests to fail intermittently.